index.js

```diff
@@ -54,7 +76,20 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+  let avgRating = 0;
+  let sumOfRatings = 0;
+  
+  if (movies.length === 0) {
+     avgRating;
+  } else { 
+    for (let i = 0; i < movies.length; i++) {
+      sumOfRatings += Number(movies[i].imdbRating)
+      avgRating = (sumOfRatings / movies.length)
```

Comment:
Here we don't need the ` () `.  Our code right now has to perform division once for every single item in the array.  We can optimize this by `return avgRating /movies.length` instead